### DISCUSSION TITLE
No puedo ingresar al repositorio de mi grupo

### DISCUSSION THREAD

**Discussion by camisanchez** [dds-jv/foro#119] [Consultas generales] [answered]
Hola, soy del grupo 21 de los viernes a la noche. Me registré en enma pero se me venció el link de la inivitaci[on y tampoco me aparece una notificación para ingresar al grupo. Quería saber si pueden agregarme. Gracias.

**Comment by gastonprieto** [comment]
Buenas!

Ahí te mande de nuevo la invitación, fijate si aceptandola te deja acceder.

**Comment by camisanchez** [comment]
Acepte la invitación que me mandaste pero sigo sin poder entrar al repositorio del grupo 21

**Comment by alejo0998** [comment]
Buenas, yo estoy en la misma situación, pertenezco al grupo 23 y curso viernes noche.

**Comment by gastonprieto** [comment]
listo @camisanchez fijate ahora

**Comment by gastonprieto** [comment]
fijate que deberías tener una invitación @alejo0998 en tu mail, y la próxima hacelo en un issue aparte esto, así identificamos a quien le falta.

**Comment by gastonprieto** [comment]
@camisanchez @alejo0998  lo pudieron resolver con eso?

**Comment by camisanchez** [comment]
Si, muchas gracias!

**Comment by alejo0998** [comment, marked as answer]
> @camisanchez @alejo0998 lo pudieron resolver con eso?

Sisi ya esta resuelto, gracias!
